mobile/Screen/MapScreen.js

```diff
@@ -29,11 +34,47 @@ function MapScreen() {
   };
 
   return (
-    <View style={styles.container}>
-      <Text>Map Screen</Text>
-      <Button title="Test" onPress={test} />
-    </View>
+    <ScrollView style={styles.scrollView}>
+      {/* <Text>Map Screen</Text>
+      <Button title="Test" onPress={test} /> */}
+{events.map((event)=>
+      <MapEvent 
+      title={event.title}
+      date={event.date}
+      location={event.location}
+      nonprofits={event.nonprofits}
+      description={event.description}
+      navigation={navigation}
+      />
+)}
+    </ScrollView>
   );
 }
 
+const events = [{
```

Comment:
If the `MapEvent` component takes in an `id` field, then each object in the `events` array needs to have that field as well